Lib/test/test_wsgiref.py

```diff
@@ -788,6 +788,18 @@ def flush(self):
             b"Hello, world!",
             written)
 
+    def testConnectionAbortedError(self):
+        class AbortingWriter:
+            def write(self, b):
+                raise ConnectionAbortedError()
+
+            def flush(self):
+                pass
+
+        environ = {"SERVER_PROTOCOL": "HTTP/1.0"}
+        h = SimpleHandler(BytesIO(), AbortingWriter(), sys.stderr, environ)
```

Comment:
Once PR #12914 is merged, exceptions will be picked up by `handle_error()`. I think we can pass a `StringIO` instance instead of `sys.stderr` and check that `stderr.getvalue()` is empty. That would address Andrew's comment in https://github.com/python/cpython/pull/9713#issuecomment-465630667.

Reply:
Done!